packages/jmix-front-generator/src/generators/react-typescript/blank-screen/index.ts

```diff
@@ -18,7 +19,7 @@ export class ReactComponentGenerator extends YeomanGenerator {
       templateDir: path.join(__dirname, 'template'),
       questions: allQuestions,
       stages: { 
-        getAnswersFromPrompt,
+        getAnswersFromPrompt: defaultGetAnswersFromPrompt,
```

Comment:
I think you can just omit `getAnswersFromPrompt` and `defaultGetAnswersFromPrompt` will be used